packages/datatrak-server/.env.example

```diff
@@ -0,0 +1,12 @@
+DB_NAME=
```

Comment:
Just double checking that you got around to adding in the env files in last pass? :)

Reply:
<img width="803" alt="image" src="https://github.com/beyondessential/tupaia/assets/69437175/4117643f-3908-451c-acf5-11408592b7b2">

Reply:
🤦